File: Java/Basics To Java/src/B/B12.java
package B;

import java.util.Scanner;

public class B12 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("je horko? a/n");
		char odpoved1 = sc.next().charAt(0);
		System.out.println("je den? a/n");
		char odpoved2 = sc.next().charAt(0);

		if ((odpoved1 == 'a' || odpoved1 == 'A') && (odpoved2 == 'a' || odpoved2 == 'A'))
			System.out.println("bež si tedy zaplavat");
		else
			System.out.println("tak nic, zùstaò doma");
	}
}

File: C#/Basic logic/B/B12.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace B
{
    internal class B12
    {

        static void Mainx(string[] args)
        {
            Console.WriteLine("je den? a/n");
            char odpoved1=char.Parse(Console.ReadLine());
            Console.WriteLine("je horko? a/n");
            char odpoved2 = char.Parse(Console.ReadLine());

            if ((odpoved1 =='a'||odpoved1=='A') && (odpoved2 =='a' || odpoved2 =='A'))
                Console.WriteLine("bež si tedy zaplavat");
            else
                Console.WriteLine("tak nic, zůstaň doma");
        }

    }
}
